policyengine_us/parameters/gov/irs/deductions/itemized/limitation/agi_rate.yaml

```diff
@@ -0,0 +1,13 @@
+description: Itemized deductions are limited to this percenatge of the excess of adjusted gross income over the applicable amount.
```

Comment:
Write other parameter descriptions/labels in a similar style
```suggestion
description: Virginia limits itemized deductions to this fraction of the excess of adjusted gross income over the applicable amount.
```

Reply:
Hi @MaxGhenis 
This is a federal level parameter. I'm wondering whether the inclusion of Virginia is unnecessary in this description?
Also for some others.

Reply:
oops, yes say "The US" instead